playbook.yml

```diff
@@ -182,6 +186,7 @@
     - win_shortcut:
         src: 'C:\ProgramData\chocolatey\bin\SQLWorkbench.exe'
         dest: 'C:\Users\Public\Desktop\SQLWorkbench\SQLWorkbench.lnk'
+        arguments: '-configDir=C:/Users/Public/.sqlworkbench'
```

Comment:
I think these will need to be `\` instead of `/`

Reply:
Fixed